# goal definition
string[] known_objects

---
#result definition
qualification/ObjectPose[] object_positions

---
# feedback
